### PR TITLE
Reduce distress when breaking backwards compatibility

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Rivet/Rivet.psm1 [hunk 1]
```diff
@@ -3,7 +3,7 @@ $Connection = New-Object Data.SqlClient.SqlConnection
 
 $RivetSchemaName = 'rivet'
 $RivetMigrationsTableName = 'Migrations'
-$RivetMigrationsTableFullName = '{0}.{1}' -f $RivetSchemaName,$RivetMigrationsTableName
+$RivetMigrationsTableFullName = "[$($RivetSchemaName)].[$($RivetMigrationsTableName)]"
```
Comment (pnstickne) [inline Rivet/Rivet.psm1:6]: It _might_ be possible to avoid the $(..) wrapping here since bounded with ].

**Review by pnstickne — APPROVED** [review]

File: Rivet/Functions/Update-Database.ps1 [hunk 2]
```diff
@@ -133,38 +131,31 @@ function Update-Database
         $byName['Include'] = $Name
     }
 
-    Get-MigrationFile -Path $Path -Configuration $Configuration @byName -ErrorAction Stop |
+    $query = 'if (object_id(''{0}'', ''U'') is not null) select ID, Name, Who, AtUtc from {0}' -f $RivetMigrationsTableFullName
```
Comment (pnstickne) [inline Rivet/Functions/Update-Database.ps1:134]: Specific warning if missing Rivet tables? Or is that elsewhere?

File: Rivet/Functions/Update-Database.ps1 [hunk 3]
```diff
@@ -133,38 +131,31 @@ function Update-Database
...
             if( $RivetSchema )
             {
                 return $true
             }
 
             if( [int64]$_.MigrationID -lt 1000000000000 )
             {
-                Write-Error ('Migration ''{0}'' has an invalid ID. IDs lower than 01000000000000 are reserved for internal use.' -f $_.FullName)
-                $stopMigrating = $true
+                Write-Error ('Migration "{0}" has an invalid ID. IDs lower than 01000000000000 are reserved for internal use.' -f $_.FullName) -ErrorAction Stop
```
Comment (pnstickne) [inline Rivet/Functions/Update-Database.ps1:152]: return after -ErrorAction Stop.

File: Rivet/Rivet.psm1 (hunk 1 above)
Comment (splatteredbits) [reply Rivet/Rivet.psm1:6]: I like using the `$()` every time so I don't have to worry about when I need to or not.

File: Rivet/Functions/Update-Database.ps1 (hunk 2 above)
Comment (splatteredbits) [reply Rivet/Functions/Update-Database.ps1:134]: This is for first-run situation where Rivet is being run for the first time so not only are there no migrations, but there are no Rivet objects in the database.

File: Rivet/Functions/Update-Database.ps1 (hunk 3 above)
Comment (splatteredbits) [reply Rivet/Functions/Update-Database.ps1:152]: That's a convention we use so it's clear that the code is returning.

**Pull request merged**